BooruSharp/Booru/Template/Gelbooru.cs

```diff
@@ -116,7 +90,7 @@ protected internal override Search.Comment.SearchResult GetCommentSearchResult(o
             return new Search.Comment.SearchResult(
                 int.Parse(elem.Attributes.GetNamedItem("id").Value),
                 int.Parse(elem.Attributes.GetNamedItem("post_id").Value),
-                creatorId.InnerText == "" ? (int?)null : int.Parse(creatorId.Value),
+                creatorId.InnerText.Length > 0 ? int.Parse(creatorId.Value) : (int?)null,
```

Comment:
Not sure why `creatorId.InnerText` was used for the check, but `creatorId.Value` was used as the actual value to parse, so I left this as is.